training.rst

```diff
@@ -0,0 +1,124 @@
+Prepare a training on OMERO
+===========================
+
+This chapter describes a setup of a training OMERO.server at 
+your institution. Further, it gives overview of the other training
+resources as well as hints for trainers about how to present OMERO during a training session.
+
+Resources
+---------
+
+- Community contributions and setups: `Github issue <https://github.com/ome/omero-guides/issues/107>`_.
+
+- `Useful scripts <https://github.com/ome/training-scripts>`_ for setting up the training server.
+
+
+Setup of scripts and OMERO.cli environment
+------------------------------------------
+
+- Clone the `training-scripts <https://github.com/ome/training-scripts>`_ repository::
+
+    $ git clone https://github.com/ome/training-scripts.git
+
+- Set up a OMERO.cli as specified under `CLI installation <https://docs.openmicroscopy.org/omero/latest/users/cli/installation.html>`_. Typically, this environment will be used on your local machine. Alternatively, you can use the OMERO.cli environment of the server you installed below (possibly, but not necessarily, on a different machine).
+
+Spin up a training server
+-------------------------
+
+System requirements for an OMERO.server are specified in the `documentation <https://omero.readthedocs.io/en/stable/sysadmins/system-requirements.html>`_. All  training servers used by the OME Team run in VMs with CentOS 7.
+
+We recommend to use the Ansible management software and use the provided Ansible playbook to install the OMERO.server for training, as this gives you the exact blueprint of the server setup used during trainings run by the OME Team.
+
+You can use the exact `training ansible playbook <https://github.com/ome/prod-playbooks/blob/master/omero/training-server/playbook.yml>`_ used by the OME Team for their training server. Alternatively, you might start with a `simpler playbook <https://github.com/ome/ansible-example-omero-addons/blob/master/playbook.yml>`_. Both of those playbooks will also install the OMERO.web Apps for you.
+
+You also have the option to follow the server installation steps in the `sysadmin documentation <https://omero.readthedocs.io/en/stable/sysadmins/unix/server-installation.html>`_. In this case, you will have to install `Apps for OMERO.web <https://www.openmicroscopy.org/omero/apps/>`_ in separate, post installation steps. We recommend to install all apps listed on the webpage except OMERO.gallery and OMERO.mapr. 
+
+In case any other installation method was used than `training ansible playbook <https://github.com/ome/prod-playbooks/blob/master/omero/training-server/playbook.yml>`_, you have to also set up the server configuration according to the `configuration block in the playbook <https://github.com/ome/prod-playbooks/blob/c72014f7f5a181d4d4daad3b86045f1c4e41a75b/omero/training-server/playbook.yml#L473>`_.
```

Comment:
```suggestion
In case any other installation method was used than `training ansible playbook <https://github.com/ome/prod-playbooks/blob/master/omero/training-server/playbook.yml>`_, you have to also set up the server configuration according to the `configuration block in the playbook <https://github.com/ome/prod-playbooks/blob/master/omero/training-server/playbook.yml#L47>`_.
```

Reply:
This whole block was removed, so the line number is not relevant now.. The `philosophical` question though remains: link to `master` or to commits ? Maybe this is related to the fact that some repos do not have frequent releases, e.g. `prod-playbooks` ?